sale_mass_mail_quotation/readme/USAGE.md

```diff
@@ -0,0 +1 @@
+Select quotations and use the sidebar action "Mass send quotation"
```

Comment:
What is the sidebar action ?

Reply:
I believe it to be this conventionally-but-wrongly-named button
![image](https://github.com/OCA/sale-workflow/assets/19902174/ffd1e0df-2ad2-4b76-8dd2-2f3cae7c0d51)

Reply:
Ok, probably time is came to name it rightly, like `action button` ?

Don't you think ?

Reply:
done